### DISCUSSION TITLE
[Internal] "Android" GitHub issue label

### DISCUSSION THREAD

**Discussion by Marcono1234** [google/gson#2716] [Ideas]
There are some issues which seem to be exclusive to Android (or even specific Android versions or devices). Maybe it would be useful to have an "android" GitHub issue label to group them? The label description could be something similar to this then:
> Issues which are exclusive to Android, and don't occur with a JDK (even if ProGuard / R8 is used)

Here is a (possibly incomplete) list of issues:
- #726
- #1242
- #1298 (maybe exclusive to Android?)
- #1343 (maybe exclusive to Android?)
- #1778 (maybe exclusive to Android, or applies also to R8 usage with JDK?)
- #2365

What do you think?

**Comment by eamonnmcmanus** [comment]
Sure, I've made that label and added it to the issues you listed.
